sdk/communication/communication-alpha-ids/README.md

```diff
@@ -0,0 +1,122 @@
+
+You can get a key and/or connection string from your Communication Services resource in the [Azure Portal][azure_portal]. You can also find the endpoint for your Communication Services resource in the [Azure Portal][azure_portal].
+
+Once you have a key, you can authenticate the `AlphaIdsClient` with any of the following methods:
+
+### Using a connection string
+
+```typescript
+import { AlphaIdsClient } from "@azure-tools/communication-alpha-ids";
+
+const connectionString = "endpoint=<endpoint>;accessKey=<accessKey>";
+const client = new AlphaIdsClient(connectionString);
+```
+
+### Using an access key with `AzureKeyCredential`
+
+If you use a key to initialize the client you will also need to provide the appropriate endpoint. You can get this endpoint from your Communication Services resource in [Azure Portal][azure_portal]. Once you have a key and endpoint, you can authenticate with the following code:
+
+```typescript
+import { AzureKeyCredential } from "@azure/core-auth";
+import { AlphaIdsClient } from "@azure-tools/communication-alpha-ids";
+
+const credential = new AzureKeyCredential("<key-from-resource>");
+const client = new AlphaIdsClient("<endpoint-from-resource>", credential);
+```
+
+### Using an Azure Active Directory Credential
+
+Connection string authentication is used in most of the examples, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][defaultazurecredential] provider shown below, or other credential providers provided with the Azure SDK, please install the [`@azure/identity`][azure_identity] package:
+
+```bash
+npm install @azure/identity
+```
+
+The [`@azure/identity`][azure_identity] package provides a variety of credential types that your application can use to do this. The [README for `@azure/identity`][azure_identity_readme] provides more details and samples to get you started.
+
+```typescript
+import { DefaultAzureCredential } from "@azure/identity";
+import { AlphaIdsClient } from "@azure-tools/communication-alpha-ids";
+
+let credential = new DefaultAzureCredential();
+const client = new AlphaIdsClient("<endpoint-from-resource>", credential);
+```
+
+## Usage
+
+The following sections provide code snippets that cover some of the common tasks using the Azure Communication Services Phone Numbers client. The scenarios that are covered here consist of:
+
+- [Get the current applied configuration](#get-the-current-applied-configuration)
+
+### Get the current applied configuration
+Use the `getConfiguration` method to obtain the current applied configuration for your resource.
+
+```typescript
+import { AlphaIdsClient } from "@azure-tools/communication-alpha-ids";
```

Comment:
could we convert this snippet to JS code instead? We could use the cjs syntax for importing.

Reply:
Changed them. Let me know if it looks good for you now